spec/models/export/submissions/extract_spec.rb

```diff
@@ -84,5 +84,22 @@
         end
       end
     end
+
+    context 'with a date range provided' do
+      let!(:previously_exported_submission) { create(:completed_submission, updated_at: 1.week.ago) }
+      let(:date_range) { 4.days.ago..1.minute.from_now }
+
+      it 'only includes submissions that were updated during the range' do
+        all_relevant = Export::Submissions::Extract.all_relevant(date_range)
+        expect(all_relevant).not_to match_array previously_exported_submission
+
+        # rubocop:disable Rails/SkipsModelValidations
+        previously_exported_submission.touch
```

Comment:
At the start of the spec this is a previously-exported submission, but this line makes it germane to the current export by settings its `updated_at` field. I can appreciate that this is to avoid creating two submissions, but Is there a way we can phrase this that doesn't change the meaning of the `let` mid-stream? Even if we just call these things `submission` and `task`?

Reply:
I agree! (I'd rewritten this a few times and forgot to update)